Formula/m-cli.rb

```diff
@@ -0,0 +1,24 @@
+class MCli < Formula
+  desc "Swiss Army Knife for Mac OS X"
+  homepage "https://github.com/rgcr/m-cli"
+  url "https://github.com/rgcr/m-cli/archive/v0.0.8.tar.gz"
+  sha256 "66956e3f21362f4baca25241dcf912c4e7aa27e329b5adb431338b52fd262a6f"
+
+  def install
+    prefix.install Dir["*"]
+    inreplace "#{prefix}/m", /^\[ -L.*|^\s+\|\| pushd.*|^popd.*/, ""
```

Comment:
What about Homebrew makes the `inreplace` necessary? Is that something we can fix?